docs/geedocs/5.3.2/sources/answer/fusionResAndProj/addFlatImageryToMercatorMapDBInGEE5.1.0.rst.txt

```diff
@@ -65,28 +65,27 @@ Add flat imagery to Mercator map databases in GEE 5.1.0
       #. Navigate to the ``ASSET_ROOT/Projects/Map``\ folder.
       #. Select the map project that you want to add to the map database
       and click **Open**.
-         The map project appears in the Map Database
+      The map project appears in the Map Database
```

Comment:
House style is to keep results of steps (vs. steps users follow) in line with the text, rather than the number.

Reply:
Probably not worth the effort of changing.

Reply:
there were rst warnings that it should be inline with the #, according to rst syntax